tensorflow/training/docker/2.5/py3/Dockerfile.cpu

```diff
@@ -140,7 +140,7 @@ RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
  && ln -s $(which ${PIP}) /usr/bin/pip
 
 RUN apt-get update && apt-get -y install protobuf-compiler \
-    && rm -rf /var/lib/apt/lists/* \ 
```

Comment:
I believe we need to update to 2.5.3 for TF - we may need a new binary as result